exporter/exporter.go

```diff
@@ -59,13 +59,14 @@ type Opts struct {
 	DiscoveringMode        bool
 	GlobalConnPool         bool
 
-	CollectAll             bool
-	EnableDBStats          bool
-	EnableDiagnosticData   bool
-	EnableReplicasetStatus bool
-	EnableTopMetrics       bool
-	EnableIndexStats       bool
-	EnableCollStats        bool
+	CollectAll                    bool
+	EnableDBStats                 bool
+	EnableDiagnosticData          bool
+	EnableReplicasetStatus        bool
+	EnableTopMetrics              bool
+	EnableIndexStats              bool
+	EnableCollStats               bool
+	EnableOverrideDescendingIndex bool
```

Comment:
This is related to metric generation, not collection. Let's add a newline to differentiate it from collector flags.